spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/embedded/TomcatWebServerFactoryCustomizer.java

```diff
@@ -89,6 +90,9 @@ public void customize(ConfigurableTomcatWebServerFactory factory) {
 		propertyMapper.from(tomcatProperties::getMaxHttpPostSize).asInt(DataSize::toBytes)
```

Comment:
This mapping and its corresponding `customizeMaxHttpPostSize` method can now be removed.